Integrations/integration-fireeye.yml

```diff
@@ -347,7 +397,7 @@ script:
     - name: frame
       description: commands.server.feReport.arguments.frame.description
     description: commands.server.feReport.description
-  - name: fe-submit-status
+  - name: fe-submit-file-status
```

Comment:
@sgosai I think it might be better to keep this as before for backwards compatibility

Reply:
@hirshbergshachar, integration has been updated.